.github/workflows/ios-e2e.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     timeout-minutes: 60
     env:
       WORKING_DIRECTORY: example
-      DEVICE: iPhone 14 Pro
+      DEVICE: iPhone 16 Pro
```

Comment:
In E2E tests, `iPhone 14 Pro` was chosen intentionally, as it has the exact pixel ratio needed

Reply:
Don't they have the same pixel ratio?

Reply:
@jakex7